core/tests/protractor/stateEditor.js

```diff
@@ -134,6 +134,30 @@ describe('State editor', function() {
     users.logout();
   });
 
+  it('should skip the customization modal for interactions having no ' +
```

Comment:
This is good, but there should also be a test for what happens when I click on an interaction after I've set it in order to try and customize it. (And in both cases, the process of changing the interaction to another one.)